src/fp.rs

```diff
@@ -655,7 +655,7 @@ impl ark_ff::Field for Fp {
 }
 
 #[cfg(test)]
-mod tests {
+pub mod tests {
```

Comment:
No idea why this is needed. It's certainly super ugly. But it's a prototype anyway ;)

Reply:
yeah prototype and i dont know why you think it's ugly - test modules are modules like the rest of the code i could very well share modules between the codebase 🤷 
But best thing would be to move it to tests.rs but i was again lazy.